data/migrations/20190716143308_createSalesTable.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("sales", table => {
+    table.increments(); // primary key
+    table.integer("sales").notNullable();
+    table
+      .integer("cars_id")
+      .unsigned()
+      .notNullable()
+      .references("id")
+      .inTable("cars");
```

Comment:
**Dig a little deeper** 👇 
If you want to learn more about foreign keys have a look at what cascade is. https://www.techonthenet.com/sql_server/foreign_keys/foreign_delete.php

With Knex you can chain `onDelete('CASCADE')` and `onUpdate('CASCADE')` to ensure that cascading takes places. It's a handy tool to have to stop the database from throwing errors when you're trying to delete or update items that are connected to other tables. :smiley_cat: